components/cdc/src/endpoint.rs

```diff
@@ -583,8 +598,20 @@ impl<T: 'static + CdcHandle<E>, E: KvEngine, S: StoreRegionMeta> Endpoint<T, E,
                 request_id,
             } => {
                 let conn = self.connections.get_mut(&conn_id).unwrap();
-                for (region, downstream) in conn.unsubscribe_request(request_id) {
-                    self.deregister_downstream(region, downstream, None);
+                for (region_id, downstream) in conn.unsubscribe_request(request_id) {
+                    let err = Some(Error::Other("region not found".into()));
```

Comment:
Nit: The "region not found" is a little confused to me, as it usually means the region is not found on this TiKV instance.

Suggest to add more appropriate message of `EventEvent` for the deregister event later.